sql/moz-fx-data-shared-prod/search_terms_derived/adm_weekly_aggregates_v1/query.sql

```diff
@@ -1,16 +1,17 @@
 SELECT
+  DATE_SUB(@submission_date, INTERVAL 6 DAY) AS start_date,
```

Comment:
I am adding a new `start_date` field here which indicates the beginning of the 7-day window represented in each row. This should help fight the misconception that we are only aggregating a single day.